composer.json

```diff
@@ -13,7 +13,7 @@
         "php": ">= 5.4.0"
     },
     "require-dev": {
-        "nette/tester": "^1.6"
+        "nette/tester": "^2.0"
```

Comment:
Shouldn't ext-calendar be added as a dev-requirement to use `easter_date()` in the test?

Reply:
First motivation for these changes are to replace this [method](https://github.com/petaak/workdays/blob/master/src/Workdays/HolidaysProvider/BaseHolidaysProvider.php#L20) by native php function. If these changes are confirmed, better way is add `ext-calendar` to require section.

Reply:
That method was created just a few weeks ago in order to avoid the dependency on ext-calendar :wink:
See 01eb1bc5193b5a4f4e0851740eef568bf7d9349b

Reply:
Ok, the method for get easter sunday is little magic. I understand, this PR does not make sence with previus goal. I undestand your notice, why add  ext-calendar to dev-requirement. I prepare it.

Reply:
Thanks 👍

Reply:
Done